Hamilton_Matthew/detail.html

```diff
@@ -0,0 +1,138 @@
+<%inherit file="app_base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="content_top">
+    <h1 class = "myh1">${ pagetitle }</h1>
+</%block>
+
+        <%block name="content_middle">
+            <ul>
+            <div class ="col-md-1">
```

Comment:
I am certain this works, but I approached this differently. How does your javascript know which image it is hovering over?

Reply:
The images have the class left_pane_image (soon to be left-pane-image) which the javascript looks at